client/management/v3/zz_generated_cloudflare_provider_config.go

```diff
@@ -1,12 +1,14 @@
 package client
 
 const (
-	CloudflareProviderConfigType          = "cloudflareProviderConfig"
-	CloudflareProviderConfigFieldAPIEmail = "apiEmail"
-	CloudflareProviderConfigFieldAPIKey   = "apiKey"
+	CloudflareProviderConfigType              = "cloudflareProviderConfig"
+	CloudflareProviderConfigFieldAPIEmail     = "apiEmail"
+	CloudflareProviderConfigFieldAPIKey       = "apiKey"
+	CloudflareProviderConfigFieldProxySetting = "proxySetting"
 )
 
 type CloudflareProviderConfig struct {
-	APIEmail string `json:"apiEmail,omitempty" yaml:"apiEmail,omitempty"`
-	APIKey   string `json:"apiKey,omitempty" yaml:"apiKey,omitempty"`
+	APIEmail     string `json:"apiEmail,omitempty" yaml:"apiEmail,omitempty"`
+	APIKey       string `json:"apiKey,omitempty" yaml:"apiKey,omitempty"`
+	ProxySetting string `json:"proxySetting,omitempty" yaml:"proxySetting,omitempty"`
```

Comment:
do we only want proxy parameter for cloudFlare? I recall seeing more params via answers, are any of them useful enough to be exposed?

Reply:
No, rest of the parameters are not for configuring cloudflare functionality. They are mainly related to the external-dns pod deployment (use k8s rbac, ownerId to know which records are created by this pod, do a full-sync Vs only add new dns records etc.)